### PR TITLE
Updated data structures notes.

### PR DESCRIPTION
**Pull request by adletaw**
Hi Andrew, I've finished my notes for the Data Structures chapter.  I renamed and deleted a couple of files so I hope that doesn't give you issues.

### CODE REVIEW

**Comment by aammd** [comment]
Merged em, love em!

hey, do you think that it makes sense for us to have 6 different notes on
the same chapter, or should we just edit each other's notes?

**Comment by adletaw** [comment]
Well, I was thinking about that too.  It is kind of a mess to have a bunch of the same notes, but I think it would be crazy to try to have everyone editing the same notes.  That sounds like a merge nightmare.  I intend to continue to edit my notes during our discussion tomorrow, for example.  So maybe for now it is best to keep them separated, and then we could compile them all into a single file with the notes of most interest for that chapter.  Even then, I think it is worth preserving the individual notes for reference/historical interest.

Basically, I'm envisioning everyone getting the opportunity to show off anything of interest in their own notes file during the discussion (or not if there is nothing).

If you want to keep it cleaner you could have sub folders for each chapter with the individualized notes, and then the compiled notes could be in your main folder.

**Comment by adletaw** [comment]
And I thought of something else as well; if we are working from the same notes, it may not be possible to get a chance at working through the exercises without seeing what someone else did already.
